force-app/main/freelanceCommunity/lwc/hot_openServiceAppointments/hot_openServiceAppointments.js

```diff
@@ -121,18 +121,27 @@ export default class Hot_openServiceAppointments extends LightningElement {
     @track serviceAppointment;
     isDetails = false;
     isSeries = false;
+    seriesRecords = [];
     showTable = true;
     goToRecordDetails(result) {
-        this.checkedServiceAppointments = this.template.querySelector('c-table').getCheckedRows();
         window.scrollTo(0, 0);
+        this.seriesRecords = [];
+        this.checkedServiceAppointments = this.template.querySelector('c-table').getCheckedRows();
         let recordId = result.detail.Id;
         this.urlStateParameterId = recordId;
         this.isDetails = this.urlStateParameterId !== '';
         for (let serviceAppointment of this.records) {
             if (recordId === serviceAppointment.Id) {
                 this.serviceAppointment = serviceAppointment;
+                this.isSeries = this.serviceAppointment.HOT_IsSerieoppdrag__c;
+            }
+        }
+        for (let serviceAppointment of this.records) {
+            if (this.serviceAppointment.HOT_Request__c === serviceAppointment.HOT_Request__c) {
+                this.seriesRecords.push(serviceAppointment);
             }
         }
+        this.isSeries = this.seriesRecords.length <= 1 ? false : true;
```

Comment:
litt tungvindt måte å gjøre det på? Kan man ikke bare bruke this.serviceAppointment.HOT_IsSerieoppdrag__c?